auth-server/src/controllers/graphql.js

```diff
@@ -0,0 +1,47 @@
+import { GraphQLSchema, GraphQLObjectType, GraphQLString, GraphQLList, GraphQLInt } from 'graphql'
```

Comment:
Just curious, any reason why this is a js file?

Reply:
renaming